SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| < 1.0.0 | :x:                |
+| 1.x.x   | :white_check_mark: |
+| 2.x.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+When disclosing security vulnerabilities please send a report to security@konghq.com
```

Comment:
Might be best to point to the source of truth about vulnerability reporting: https://docs.konghq.com/enterprise/2.4.x/kong-security-update-process/#reporting-a-vulnerability

Reply:
:+1: e784d43589a3ec51afb595ad661b714aca236527